Documentation: Contributing Website is out of date

The instructions for building and testing are out of date: https://mesh-for-data.github.io/mesh-for-data/v0.1/contribute/build-test/ @ronenkat 

The following steps listed in the documentation do not exist and thus cannot be run.
```
make -C manager deploy-crd
make -C manager deploy_it
```

In addition, I have problems running several of the commands (which may or may not be specific to my environment):
1) make kind
port 5001 is taken by my cisco vpn even when it is not running, so I kind-registry-proxy doesn't run
```
docker push ghcr.io/mesh-for-data/manager:latest
The push refers to repository [ghcr.io/mesh-for-data/manager]
50282b699498: Preparing 
84ff92691f90: Preparing 
42a3027eaac1: Preparing 
denied: unauthenticated: User cannot be authenticated with the token provided.
make[1]: *** [docker-push] Error 1
make: *** [docker-push] Error 2
```


2) "make docker" does not run successfully.  I get the following error.  
```
docker push ghcr.io/mesh-for-data/manager:latest
The push refers to repository [ghcr.io/mesh-for-data/manager]
50282b699498: Preparing 
84ff92691f90: Preparing 
42a3027eaac1: Preparing 
denied: unauthenticated: **User cannot be authenticated with the token provided**.
make[1]: *** [docker-push] Error 1
make: *** [docker-push] Error 2

```

--- a/hack/tools/create_kind.sh
+++ b/hack/tools/create_kind.sh
@@ -8,7 +8,7 @@ op=$1
 
 source ./common.sh
 
-K8S_VERSION=${K8S_VERSION:-v1.18.19@sha256:7af1492e19b3192a79f606e43c35fb741e520d195f96399284515f077b3b622c}
+K8S_VERSION=${K8S_VERSION:-v1.19.11@sha256:07db187ae84b4b7de440a73886f008cf903fcf5764ba8106a9fd5243d6f32729}
 : ${KUBE_NAMESPACE:=fybrik-system}
 : ${KEEP_PROXY:=true}
 
@@ -79,7 +79,8 @@ install_nginx_ingress() {
         kubectl create ns "$KUBE_NAMESPACE" || true
 
         echo Install ingress-nginx
-        kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/provider/kind/deploy.yaml
+        # Using v1.0.3 because of https://github.com/kubernetes/ingress-nginx/issues/7810
+        kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.0.3/deploy/static/provider/kind/deploy.yaml
         kubectl wait --namespace ingress-nginx \
           --for=condition=ready pod \
           --selector=app.kubernetes.io/component=controller \

--- a/site/docs/contribute/build-test.md
+++ b/site/docs/contribute/build-test.md
@@ -38,8 +38,6 @@ USE_EXISTING_CONTROLLER=true NO_SIMULATED_PROGRESS=true USE_EXISTING_CLUSTER=tru
 
 ## Running integration tests
 
-### Running in one step
-
 With the following you will then setup a kind cluster with the local registry,
 build and push current docker images and finally run the integration
 tests on it:
@@ -48,50 +46,7 @@ tests on it:
 make run-integration-tests
 ```
 
-### Running step by step
-
-It is also possible to call the commands step by step, which sometimes is
-useful if you want to only repeat a specific step which failed without having
-to rerun  the entire sequence
-
-```bash
-# use the local kind registry
-export DOCKER_HOSTNAME=kind-registry:5000
-export DOCKER_NAMESPACE=fybrik-system
-
-# build a local kind cluser
-make kind
-
-# deploy the the cluster 3rd party such as cert-manager and vault
-make cluster-prepare
-
-# build all docker images and push them to the local registry
-make docker
-
-# build the mock/test docker images and push them to local registry
-make -C test/services docker-build docker-push
-
-# wait until cluster-prepare setup really completed
-make cluster-prepare-wait
-
-# deploy the fybrik CRDs to the kind cluster
-make -C manager deploy-crd
-
-# deploy fybrik manager to the kind cluster
-make -C manager deploy_it
-
-# wait until manager is ready
-make -C manager wait_for_manager
-
-# configure Vault
-make configure-vault
-
-# build and push helm charts to the local registry
-make helm
-
-# actually run the integration tests
-make -C manager run-integration-tests
-```
+You can run `make kind-cleanup` to delete the created clusters when you're done.
 
 ## Building in a multi cluster environment
 
@@ -103,3 +58,5 @@ Two kind clusters that share the same kind-registry can be set up using:
 ```bash
 make kind-setup-multi
 ``` 
+
+You can run `make kind-cleanup` to delete the created clusters when you're done.

--- a/site/docs/contribute/build-test.md
+++ b/site/docs/contribute/build-test.md
@@ -46,6 +46,9 @@ tests on it:
 make run-integration-tests
 ```
 
+It is sometimes useful to to call the integration test commands step by step, e.g., if you want to only repeat a specific step which failed without having to rerun the entire sequence. You can find the commands of the `run-integration-tests` target in the [Makefile](https://github.com/fybrik/fybrik/blob/master/Makefile).
+
+
 You can run `make kind-cleanup` to delete the created clusters when you're done.
 
 ## Building in a multi cluster environment

--- a/site/docs/reference/connectors.md
+++ b/site/docs/reference/connectors.md
@@ -39,38 +39,6 @@
 
 
 
-<a name="data_catalog_request.proto"></a>
-<p align="right"><a href="#top">Top</a></p>
-
-## data_catalog_request.proto
-
-
- <!-- end services -->
-
-
-<a name="connectors.CatalogDatasetRequest"></a>
-
-### CatalogDatasetRequest
-
-
-
-| Field | Type | Label | Description |
-| ----- | ---- | ----- | ----------- |
-| credential_path | [string](#string) |  | link to vault plugin for reading k8s secret with user credentials |
-| dataset_id | [string](#string) |  | identifier of asset - always needed. JSON expected. Interpreted by the Connector, can contain any additional information as part of JSON |
-
-
-
-
-
- <!-- end messages -->
-
- <!-- end enums -->
-
- <!-- end HasExtensions -->
-
-
-
 <a name="data_catalog_response.proto"></a>
 <p align="right"><a href="#top">Top</a></p>
 
@@ -103,6 +71,316 @@
 
 
 
+<a name="policy_manager_response.proto"></a>
+<p align="right"><a href="#top">Top</a></p>
+
+## policy_manager_response.proto
+
+
+ <!-- end services -->
+
+
+<a name="connectors.ComponentVersion"></a>
+
+### ComponentVersion
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| name | [string](#string) |  |  |
+| id | [string](#string) |  |  |
+| version | [string](#string) |  |  |
+
+
+
+
+
+
+<a name="connectors.DatasetDecision"></a>
+
+### DatasetDecision
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| dataset | [DatasetIdentifier](#connectors.DatasetIdentifier) |  |  |
+| decisions | [OperationDecision](#connectors.OperationDecision) | repeated |  |
+
+
+
+
+
+
+<a name="connectors.EnforcementAction"></a>
+
+### EnforcementAction
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| name | [string](#string) |  |  |
+| id | [string](#string) |  |  |
+| level | [EnforcementAction.EnforcementActionLevel](#connectors.EnforcementAction.EnforcementActionLevel) |  |  |
+| args | [EnforcementAction.ArgsEntry](#connectors.EnforcementAction.ArgsEntry) | repeated |  |
+
+
+
+
+
+
+<a name="connectors.EnforcementAction.ArgsEntry"></a>
+
+### EnforcementAction.ArgsEntry
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| key | [string](#string) |  |  |
+| value | [string](#string) |  |  |
+
+
+
+
+
+
+<a name="connectors.OperationDecision"></a>
+
+### OperationDecision
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| operation | [AccessOperation](#connectors.AccessOperation) |  |  |
+| enforcement_actions | [EnforcementAction](#connectors.EnforcementAction) | repeated |  |
+| used_policies | [Policy](#connectors.Policy) | repeated |  |
+
+
+
+
+
+
+<a name="connectors.PoliciesDecisions"></a>
+
+### PoliciesDecisions
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| component_versions | [ComponentVersion](#connectors.ComponentVersion) | repeated |  |
+| dataset_decisions | [DatasetDecision](#connectors.DatasetDecision) | repeated | one per dataset |
+| general_decisions | [OperationDecision](#connectors.OperationDecision) | repeated |  |
+
+
+
+
+
+
+<a name="connectors.Policy"></a>
+
+### Policy
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| id | [string](#string) |  |  |
+| name | [string](#string) |  |  |
+| description | [string](#string) |  |  |
+| type | [string](#string) |  |  |
+| hierarchy | [string](#string) | repeated |  |
+
+
+
+
+
+ <!-- end messages -->
+
+
+<a name="connectors.EnforcementAction.EnforcementActionLevel"></a>
+
+### EnforcementAction.EnforcementActionLevel
+
+
+| Name | Number | Description |
+| ---- | ------ | ----------- |
+| UNKNOWN | 0 |  |
+| DATASET | 1 |  |
+| COLUMN | 2 |  |
+| ROW | 3 |  |
+| CELL | 4 |  |
+
+
+ <!-- end enums -->
+
+ <!-- end HasExtensions -->
+
+
+
+<a name="data_catalog_request.proto"></a>
+<p align="right"><a href="#top">Top</a></p>
+
+## data_catalog_request.proto
+
+
+ <!-- end services -->
+
+
+<a name="connectors.CatalogDatasetRequest"></a>
+
+### CatalogDatasetRequest
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| credential_path | [string](#string) |  | link to vault plugin for reading k8s secret with user credentials |
+| dataset_id | [string](#string) |  | identifier of asset - always needed. JSON expected. Interpreted by the Connector, can contain any additional information as part of JSON |
+
+
+
+
+
+ <!-- end messages -->
+
+ <!-- end enums -->
+
+ <!-- end HasExtensions -->
+
+
+
+<a name="policy_manager_request.proto"></a>
+<p align="right"><a href="#top">Top</a></p>
+
+## policy_manager_request.proto
+
+
+ <!-- end services -->
+
+
+<a name="connectors.AccessOperation"></a>
+
+### AccessOperation
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| type | [AccessOperation.AccessType](#connectors.AccessOperation.AccessType) |  |  |
+| destination | [string](#string) |  | Destination for transfer or write. |
+
+
+
+
+
+
+<a name="connectors.ApplicationContext"></a>
+
+### ApplicationContext
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| credential_path | [string](#string) |  | link to vault plugin for reading k8s secret with user credentials |
+| app_info | [ApplicationDetails](#connectors.ApplicationDetails) |  |  |
+| datasets | [DatasetContext](#connectors.DatasetContext) | repeated |  |
+| general_operations | [AccessOperation](#connectors.AccessOperation) | repeated |  |
+
+
+
+
+
+
+<a name="connectors.ApplicationDetails"></a>
+
+### ApplicationDetails
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| processing_geography | [string](#string) |  |  |
+| properties | [ApplicationDetails.PropertiesEntry](#connectors.ApplicationDetails.PropertiesEntry) | repeated |  |
+
+
+
+
+
+
+<a name="connectors.ApplicationDetails.PropertiesEntry"></a>
+
+### ApplicationDetails.PropertiesEntry
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| key | [string](#string) |  |  |
+| value | [string](#string) |  |  |
+
+
+
+
+
+
+<a name="connectors.DatasetContext"></a>
+
+### DatasetContext
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| dataset | [DatasetIdentifier](#connectors.DatasetIdentifier) |  |  |
+| operation | [AccessOperation](#connectors.AccessOperation) |  |  |
+
+
+
+
+
+
+<a name="connectors.DatasetIdentifier"></a>
+
+### DatasetIdentifier
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| dataset_id | [string](#string) |  | identifier of asset - always needed. JSON expected. Interpreted by the Connector, can contain any additional information as part of JSON |
+
+
+
+
+
+ <!-- end messages -->
+
+
+<a name="connectors.AccessOperation.AccessType"></a>
+
+### AccessOperation.AccessType
+
+
+| Name | Number | Description |
+| ---- | ------ | ----------- |
+| UNKNOWN | 0 |  |
+| READ | 1 |  |
+| COPY | 2 |  |
+| WRITE | 3 |  |
+
+
+ <!-- end enums -->
+
+ <!-- end HasExtensions -->
+
+
+
 <a name="data_catalog_service.proto"></a>
 <p align="right"><a href="#top">Top</a></p>
 
@@ -121,6 +399,63 @@
 | RegisterDatasetInfo | [RegisterAssetRequest](#connectors.RegisterAssetRequest) | [RegisterAssetResponse](#connectors.RegisterAssetResponse) |  |
 
  <!-- end services -->
+
+ <!-- end messages -->
+
+ <!-- end enums -->
+
+ <!-- end HasExtensions -->
+
+
+
+<a name="policy_manager_service.proto"></a>
+<p align="right"><a href="#top">Top</a></p>
+
+## policy_manager_service.proto
+
+
+
+<a name="connectors.PolicyManagerService"></a>
+
+### PolicyManagerService
+
+
+| Method Name | Request Type | Response Type | Description |
+| ----------- | ------------ | ------------- | ------------|
+| GetPoliciesDecisions | [ApplicationContext](#connectors.ApplicationContext) | [PoliciesDecisions](#connectors.PoliciesDecisions) |  |
+
+ <!-- end services -->
+
+ <!-- end messages -->
+
+ <!-- end enums -->
+
+ <!-- end HasExtensions -->
+
+
+
+<a name="register_asset_response.proto"></a>
+<p align="right"><a href="#top">Top</a></p>
+
+## register_asset_response.proto
+
+
+ <!-- end services -->
+
+
+<a name="connectors.RegisterAssetResponse"></a>
+
+### RegisterAssetResponse
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| asset_id | [string](#string) |  | Returns the id of the new asset registered in a catalog |
+
+
+
+
 
  <!-- end messages -->
 
@@ -358,310 +693,6 @@
 
 
 
-<a name="policy_manager_request.proto"></a>
-<p align="right"><a href="#top">Top</a></p>
-
-## policy_manager_request.proto
-
-
- <!-- end services -->
-
-
-<a name="connectors.AccessOperation"></a>
-
-### AccessOperation
-
-
-
-| Field | Type | Label | Description |
-| ----- | ---- | ----- | ----------- |
-| type | [AccessOperation.AccessType](#connectors.AccessOperation.AccessType) |  |  |
-| destination | [string](#string) |  | Destination for transfer or write. |
-
-
-
-
-
-
-<a name="connectors.ApplicationContext"></a>
-
-### ApplicationContext
-
-
-
-| Field | Type | Label | Description |
-| ----- | ---- | ----- | ----------- |
-| credential_path | [string](#string) |  | link to vault plugin for reading k8s secret with user credentials |
-| app_info | [ApplicationDetails](#connectors.ApplicationDetails) |  |  |
-| datasets | [DatasetContext](#connectors.DatasetContext) | repeated |  |
-| general_operations | [AccessOperation](#connectors.AccessOperation) | repeated |  |
-
-
-
-
-
-
-<a name="connectors.ApplicationDetails"></a>
-
-### ApplicationDetails
-
-
-
-| Field | Type | Label | Description |
-| ----- | ---- | ----- | ----------- |
-| processing_geography | [string](#string) |  |  |
-| properties | [ApplicationDetails.PropertiesEntry](#connectors.ApplicationDetails.PropertiesEntry) | repeated |  |
-
-
-
-
-
-
-<a name="connectors.ApplicationDetails.PropertiesEntry"></a>
-
-### ApplicationDetails.PropertiesEntry
-
-
-
-| Field | Type | Label | Description |
-| ----- | ---- | ----- | ----------- |
-| key | [string](#string) |  |  |
-| value | [string](#string) |  |  |
-
-
-
-
-
-
-<a name="connectors.DatasetContext"></a>
-
-### DatasetContext
-
-
-
-| Field | Type | Label | Description |
-| ----- | ---- | ----- | ----------- |
-| dataset | [DatasetIdentifier](#connectors.DatasetIdentifier) |  |  |
-| operation | [AccessOperation](#connectors.AccessOperation) |  |  |
-
-
-
-
-
-
-<a name="connectors.DatasetIdentifier"></a>
-
-### DatasetIdentifier
-
-
-
-| Field | Type | Label | Description |
-| ----- | ---- | ----- | ----------- |
-| dataset_id | [string](#string) |  | identifier of asset - always needed. JSON expected. Interpreted by the Connector, can contain any additional information as part of JSON |
-
-
-
-
-
- <!-- end messages -->
-
-
-<a name="connectors.AccessOperation.AccessType"></a>
-
-### AccessOperation.AccessType
-
-
-| Name | Number | Description |
-| ---- | ------ | ----------- |
-| UNKNOWN | 0 |  |
-| READ | 1 |  |
-| COPY | 2 |  |
-| WRITE | 3 |  |
-
-
- <!-- end enums -->
-
- <!-- end HasExtensions -->
-
-
-
-<a name="policy_manager_response.proto"></a>
-<p align="right"><a href="#top">Top</a></p>
-
-## policy_manager_response.proto
-
-
- <!-- end services -->
-
-
-<a name="connectors.ComponentVersion"></a>
-
-### ComponentVersion
-
-
-
-| Field | Type | Label | Description |
-| ----- | ---- | ----- | ----------- |
-| name | [string](#string) |  |  |
-| id | [string](#string) |  |  |
-| version | [string](#string) |  |  |
-
-
-
-
-
-
-<a name="connectors.DatasetDecision"></a>
-
-### DatasetDecision
-
-
-
-| Field | Type | Label | Description |
-| ----- | ---- | ----- | ----------- |
-| dataset | [DatasetIdentifier](#connectors.DatasetIdentifier) |  |  |
-| decisions | [OperationDecision](#connectors.OperationDecision) | repeated |  |
-
-
-
-
-
-
-<a name="connectors.EnforcementAction"></a>
-
-### EnforcementAction
-
-
-
-| Field | Type | Label | Description |
-| ----- | ---- | ----- | ----------- |
-| name | [string](#string) |  |  |
-| id | [string](#string) |  |  |
-| level | [EnforcementAction.EnforcementActionLevel](#connectors.EnforcementAction.EnforcementActionLevel) |  |  |
-| args | [EnforcementAction.ArgsEntry](#connectors.EnforcementAction.ArgsEntry) | repeated |  |
-
-
-
-
-
-
-<a name="connectors.EnforcementAction.ArgsEntry"></a>
-
-### EnforcementAction.ArgsEntry
-
-
-
-| Field | Type | Label | Description |
-| ----- | ---- | ----- | ----------- |
-| key | [string](#string) |  |  |
-| value | [string](#string) |  |  |
-
-
-
-
-
-
-<a name="connectors.OperationDecision"></a>
-
-### OperationDecision
-
-
-
-| Field | Type | Label | Description |
-| ----- | ---- | ----- | ----------- |
-| operation | [AccessOperation](#connectors.AccessOperation) |  |  |
-| enforcement_actions | [EnforcementAction](#connectors.EnforcementAction) | repeated |  |
-| used_policies | [Policy](#connectors.Policy) | repeated |  |
-
-
-
-
-
-
-<a name="connectors.PoliciesDecisions"></a>
-
-### PoliciesDecisions
-
-
-
-| Field | Type | Label | Description |
-| ----- | ---- | ----- | ----------- |
-| component_versions | [ComponentVersion](#connectors.ComponentVersion) | repeated |  |
-| dataset_decisions | [DatasetDecision](#connectors.DatasetDecision) | repeated | one per dataset |
-| general_decisions | [OperationDecision](#connectors.OperationDecision) | repeated |  |
-
-
-
-
-
-
-<a name="connectors.Policy"></a>
-
-### Policy
-
-
-
-| Field | Type | Label | Description |
-| ----- | ---- | ----- | ----------- |
-| id | [string](#string) |  |  |
-| name | [string](#string) |  |  |
-| description | [string](#string) |  |  |
-| type | [string](#string) |  |  |
-| hierarchy | [string](#string) | repeated |  |
-
-
-
-
-
- <!-- end messages -->
-
-
-<a name="connectors.EnforcementAction.EnforcementActionLevel"></a>
-
-### EnforcementAction.EnforcementActionLevel
-
-
-| Name | Number | Description |
-| ---- | ------ | ----------- |
-| UNKNOWN | 0 |  |
-| DATASET | 1 |  |
-| COLUMN | 2 |  |
-| ROW | 3 |  |
-| CELL | 4 |  |
-
-
- <!-- end enums -->
-
- <!-- end HasExtensions -->
-
-
-
-<a name="policy_manager_service.proto"></a>
-<p align="right"><a href="#top">Top</a></p>
-
-## policy_manager_service.proto
-
-
-
-<a name="connectors.PolicyManagerService"></a>
-
-### PolicyManagerService
-
-
-| Method Name | Request Type | Response Type | Description |
-| ----------- | ------------ | ------------- | ------------|
-| GetPoliciesDecisions | [ApplicationContext](#connectors.ApplicationContext) | [PoliciesDecisions](#connectors.PoliciesDecisions) |  |
-
- <!-- end services -->
-
- <!-- end messages -->
-
- <!-- end enums -->
-
- <!-- end HasExtensions -->
-
-
-
 <a name="register_asset_request.proto"></a>
 <p align="right"><a href="#top">Top</a></p>
 
@@ -683,37 +714,6 @@
 | dataset_details | [DatasetDetails](#connectors.DatasetDetails) |  |  |
 | destination_catalog_id | [string](#string) |  |  |
 | credential_path | [string](#string) |  | link to vault plugin for reading k8s secret with user credentials |
-
-
-
-
-
- <!-- end messages -->
-
- <!-- end enums -->
-
- <!-- end HasExtensions -->
-
-
-
-<a name="register_asset_response.proto"></a>
-<p align="right"><a href="#top">Top</a></p>
-
-## register_asset_response.proto
-
-
- <!-- end services -->
-
-
-<a name="connectors.RegisterAssetResponse"></a>
-
-### RegisterAssetResponse
-
-
-
-| Field | Type | Label | Description |
-| ----- | ---- | ----- | ----------- |
-| asset_id | [string](#string) |  | Returns the id of the new asset registered in a catalog |
 
 
 

